tests/OnlineSales.Tests/SimpleTableTests.cs

```diff
@@ -167,6 +169,126 @@ public async Task InvalidLimit(string filter, int dataCount)
         await GetTest($"{this.itemsUrl}?{filter}", HttpStatusCode.BadRequest);
     }
 
+    [Theory]
```

Comment:
@Exmirai I would like to see some kind of error messages to differentiate one error from another. Is it possible? Now I can see all these filters are invalid, but can I somehow understand why have you selected exactly these filters and not the others? The amount of potential options is infinite here. Why do you test exactly these formats? Will the client get different or the same error for all these cases?